spec/lang/types.md

```diff
@@ -220,13 +219,13 @@ impl TupleHeadLayout {
         let align = capped_tail_align.max(self.align);
         let tail_offset = self.tail_offset(tail_align);
         let end = tail_offset + tail_size;
-        let size = Size::from_bytes(end.bytes().next_multiple_of(align.bytes())).unwrap();
+        let size = end.align_to(align);
         (size, align)
     }
 
     /// Returns the size and alignment of the tuple when there is no tail.
     pub fn head_size_and_align(self) -> (Size, Align) {
-        self.full_size_and_align(Size::ZERO, Align::ONE)
+        (self.end.align_to(self.align), self.align)
```

Comment:
Please add a comment saying this is the same as `full_size_and_align(ZERO, ONE)`.